autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -4,18 +4,89 @@
 from django.core.management import base
 from scraper.banner_requests import BannerRequests
 from scraper.models import Course, Instructor, Section, Meeting
+from course import generate_course_id
 
-def parse_section(section):
-    " parse_course() must be done first, and pass the data for each section here "
+def parse_section(course):
+    #gets values to create object
+    section_id = ''
```

Comment:
Couldn't find a section id labeled in the data. It might be id but I'm not sure. In that case, section_id = course['id']